pages/home.jsx

```diff
@@ -295,11 +303,13 @@ var HomePage = React.createClass({
               linkTo="activities"
               imgSrc="/img/pages/home/svg/icon-teachanactivity.svg"
               head="Teach an Activity"
+              onClick={this.handleTeachBtnClick}
             />
             <IconButton
               linkTo="mozilla-clubs"
               imgSrc="/img/pages/home/svg/icon-startamozillaclub.svg"
               head="Start A Mozilla Club"
+              onClick={this.handleClubBtnClick}
```

Comment:
Yup yup they do! Thanks for raising this concern. Once this gets pushed out to staging we can double check again!  :grin: 

(currently we are having trouble deploying staging, so I'll gonna keep this PR open until we are able to push out updates)